velox/exec/Task.h

```diff
@@ -33,6 +33,16 @@ class CrossJoinBridge;
 
 class Task : public std::enable_shared_from_this<Task> {
  public:
+  class JoinBridgeTranslator {
+   public:
+    virtual ~JoinBridgeTranslator() = default;
+
+    // Translates plan node to join bridge. Returns nullptr if the plan node
+    // cannot be handled by this factory.
+    virtual std::shared_ptr<JoinBridge> translate(
```

Comment:
It seems like it would be cleaner to move this method to PlanNodeTranslator in Operator.h

Reply:
Agree.